samples/msal-browser-samples/VanillaJSTestApp2.0/app/inMemoryStorage/authConfig.js

```diff
@@ -0,0 +1,55 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
```

Comment:
If the purpose of adding this sample is for E2E testing can we just add the tests to the customizable-e2e-test?
Would like to try to reduce the number of samples we have